subsys/bluetooth/controller/ll_sw/nordic/lll/lll_peripheral_iso.c

```diff
@@ -530,7 +530,21 @@ static void isr_rx(void *param)
 
 	/* FIXME: Consider Flush Timeout when resetting current burst number */
 	if (!has_tx) {
+		uint8_t bn;
+
 		has_tx = 1U;
+
+		/* Adjust sn when flushing Tx. Stop at sn != nesn, hence
+		 * (bn < cis_lll->tx.bn).
+		 */
+		bn = bn_tx;
+		while (bn < cis_lll->tx.bn) {
+			cis_lll->sn++;
+
+			bn++;
+		}
```

Comment:
Ditto. 

Perhaps since we are doing the exact same thing, this action could even be put into a function or macro.